ui/app/adapters/config-ui/message.js

```diff
@@ -4,5 +4,26 @@
  */
 
 import ApplicationAdapter from '../application';
+import { encodePath } from 'vault/utils/path-encoding-helpers';
 
-export default class MessageAdapter extends ApplicationAdapter {}
+export default class MessageAdapter extends ApplicationAdapter {
+  getCustomMessagesUrl(id) {
+    let url = `${this.buildURL()}/config/ui/custom-messages`;
+
+    if (id) {
+      url = url + '/' + encodePath(id);
+    }
+    return url;
+  }
+
+  async query(store, type, query) {
+    const { authenticated } = query;
+
+    return await this.ajax(this.getCustomMessagesUrl(), 'GET', { data: { authenticated } });
```

Comment:
unless you're doing something with the returned response, I don't believe you need this to be async.